cli.js

```diff
@@ -159,6 +159,16 @@ Usage: node cli.js -c <crawler> -d <device> -w <viewport> -u <url> OPTIONS`,
     }
     return nameEmail;
   })
+  .coerce('f', option => {
+    if (option !== 'true' && value !== 'false') {
```

Comment:
Where does the `value` come from? When i try running the cli with an invalid option `-f truee`, it throws me an error `value is not defined`